src/vm/appdomain.cpp

```diff
@@ -6515,6 +6527,8 @@ HRESULT RuntimeInvokeHostAssemblyResolver(INT_PTR pManagedAssemblyLoadContextToB
         {
             hr = COR_E_FILENOTFOUND;
         }
+
+        tracer.SetResult(hr, static_cast<BINDER_SPACE::Assembly *>(pResolvedAssembly));
```

Comment:
We're missing the HRs from the explicit throws above - note that recently I had somebody ask me about a case where one of the explicit throws above was hit - would be very nice to have a trace about it happening in relationship to the binder state.

Reply:
Throws are being captured now.  (Change going to the runtime repo soon.)